lib/callbacks.js

```diff
@@ -48,46 +40,42 @@ const once = (
   return wrap;
 };
 
-const unsafeCallback = (
-  // Extract callback function
-  // It's unsafe: may return null, allows multiple calls
-  args // array, arguments
-  // Returns: function, callback or null
-  // Hint: previous name: `common.cbUnsafe` (deprecated)
-  // Hint: another alias: `common.extractCallback` (deprecated)
-) => {
+// Extract callback function
+// It's unsafe: may return null, allows multiple calls
+//   args - array, arguments
+// Returns: function, callback or null
+// Hint: previous name: `common.cbUnsafe` (deprecated)
+// Hint: another alias: `common.extractCallback` (deprecated)
+const unsafeCallback = args => {
   const callback = last(args);
   if (typeof callback === 'function') return args.pop();
   return null;
 };
 
-const safeCallback = (
-  // Extract callback
-  args // array, arguments
-  // Returns: function, wrapped callback or common.emptiness if there is no one
-  // Hint: previous name: `cbExtract` (deprecated)
-) => {
+// Extract callback
+//   args - array, arguments
+// Returns: function, wrapped callback or common.emptiness if there is no one
+// Hint: previous name: `cbExtract` (deprecated)
+const safeCallback = args => {
   const callback = last(args);
   if (typeof callback === 'function') return args.pop();
   return emptiness;
 };
 
-const requiredCallback = (
-  // Extract callback
-  args // array, arguments
-  // Returns: function or throw TypeError if there is no callback
-) => {
+// Extract callback
+//   args - array, arguments
+// Returns: function or throw TypeError if there is no callback
```

Comment:
Should we consider adding 'Throws' clause for such cases?
/cc @SemenchenkoVitaliy @belochub @nechaido @tshemsedinov wdyt?

Reply:
Don't think so. Now we need quick decisions and results, have no time to prepare whole special language for commenting.

Reply:
It's the same syntax as `Deprecate` + `Argument` so I don't think it will be complicated.
`Throws: Error, when there is no callback`
`Throws: type, description`